CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 - Implemented stringer methods for pool.Role (#405).
 
 ### Changed
+- More informative request canceling: log the probable reason for unexpected request ID
+  and add request ID info to context done error message (#407).
```

Comment:
```suggestion
### Changed

- More informative request canceling: log the probable reason for unexpected request ID
  and add request ID info to context done error message (#407).
```

Reply:
Done: I fixed and squashed commits into one